### DISCUSSION TITLE
Nop node as a signal meter...?

### DISCUSSION THREAD

**Discussion by loatus** [maximecb/noisecraft#87] [Ideas]
maybe the Nop node could also double as a signal meter displaying the varying signal as a constantly updating number with say two decimal places. the  numbers further to the right of decimal point would change more quickly . it would give one a quick idea of the signal range and would be also useful for me to know the frequencies being ouput from the monoseq ... just an idea.. if it doesn't add to the compute load too much.

**Comment by maximecb** [comment]
The purpose of the Nop node is just to act as way to gather a lot of cables in a single point, and I'd rather avoid adding any other functionality to it (nop is short for no-op).

However, I've been thinking that it might make sense to have an "auto range" mode on the `Scope` node, enabled via a checkbox in the node parameters. In that case, it might make sense to display the min and max values on the scope.

**Comment by loatus** [comment]
yeah that sounds like a good idea with the auto range on the scope and the min max value's would be better suited to that node also.  :)
